docs/src/main/sphinx/connector/hive.rst

```diff
@@ -474,132 +319,70 @@ Hive configuration properties
       - Maximum number of partitions to drop in a single query.
       - 100,000
 
-ORC format configuration properties
------------------------------------
-
-The following properties are used to configure the read and write operations
-with ORC files performed by the Hive connector.
-
-.. list-table:: ORC format configuration properties
-    :widths: 30, 50, 20
-    :header-rows: 1
+Metastores
+----------
 
-    * - Property Name
-      - Description
-      - Default
-    * - ``hive.orc.time-zone``
-      - Sets the default time zone for legacy ORC files that did not declare a
-        time zone.
-      - JVM default
-    * - ``hive.orc.use-column-names``
-      - Access ORC columns by name. By default, columns in ORC files are
-        accessed by their ordinal position in the Hive table definition. The
-        equivalent catalog session property is ``orc_use_column_names``.
-      - ``false``
-    * - ``hive.orc.bloom-filters.enabled``
-      - Enable bloom filters for predicate pushdown.
-      - ``false``
+The Hive connector supports the use of the Hive Metastore Service (HMS) and AWS
+Glue catalog.
```

Comment:
The previous wording was "Glue Data Catalog", I think that was more consistent with what the feature is called in the AWS documentation, and helps separate it from the other ways we use the word "catalog"